tests/test_enum.py

```diff
@@ -238,13 +238,13 @@ def test_duplicate_enum_name():
 
 def test_char_underlying_enum():  # Issue #1331/PR #1334:
     assert type(m.ScopedCharEnum.Positive.__int__()) is int
-    assert int(m.ScopedChar16Enum.Zero) == 0  # int() call should successfully return
```

Comment:
The intent of this test was that when `__int__()` returns a non-int, `int()` call will throw a error:
```
>>> Player.black.__int__()
'\x01'

>>> int(Player.black)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __int__ returned non-int (type str)
```
So I wrote this call in the test and expected it to successfully return.

Reply:
Thanks. I think that is clear just from the test code itself.